<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QWidget" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="SmoothScrollArea" name="SmoothScrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: #f7f9fc;
border: none;
padding: 15px;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>677</width>
        <height>622</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>25</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="visual_setting">
         <property name="styleSheet">
          <string notr="true">#visual_setting {
	background-color: #eff4f9;
	border-radius: 15px;
}

#visual_setting:hover {
	border: 1px solid #009faa;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="BodyLabel" name="BodyLabel">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">FluentLabelBase {
    color: black;
	background-color: #eff4f9;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#ff000000}</string>
              </property>
              <property name="text">
               <string>主题</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="pixelFontSize" stdset="0">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="ComboBox" name="choice_theme"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="BodyLabel" name="BodyLabel_6">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">FluentLabelBase {
    color: black;
	background-color: #eff4f9;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#ff000000}</string>
              </property>
              <property name="text">
               <string>字体大小</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="pixelFontSize" stdset="0">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="SpinBox" name="font_size_setting"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="HorizontalSeparator" name="HorizontalSeparator"/>
       </item>
       <item>
        <widget class="QFrame" name="model_setting">
         <property name="styleSheet">
          <string notr="true">#model_setting {
	background-color: #eff4f9;
	border-radius: 15px;
}

#model_setting:hover {
	border: 1px solid #009faa;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="BodyLabel" name="BodyLabel_4">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">FluentLabelBase {
    color: black;
	background-color: #eff4f9;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#ff000000}</string>
              </property>
              <property name="text">
               <string>回复长度限制</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="pixelFontSize" stdset="0">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="SpinBox" name="limit_token_spin_box"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="BodyLabel" name="BodyLabel_3">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">FluentLabelBase {
    color: black;
	background-color: #eff4f9;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#ff000000}</string>
              </property>
              <property name="text">
               <string>top-k(随机性)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="pixelFontSize" stdset="0">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="SpinBox" name="top_k_spin_box"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="BodyLabel" name="BodyLabel_2">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">FluentLabelBase {
    color: black;
	background-color: #eff4f9;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#ff000000}</string>
              </property>
              <property name="text">
               <string>temperature(灵活度)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="pixelFontSize" stdset="0">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="DoubleSpinBox" name="temperature_spin_box"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="BodyLabel" name="BodyLabel_5">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>12</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">FluentLabelBase {
    color: black;
	background-color: #eff4f9;
}

HyperlinkLabel {
    color: #009faa;
    border: none;
    background-color: transparent;
    text-align: left;
    padding: 0;
    margin: 0;
}

HyperlinkLabel[underline=true] {
    text-decoration: underline;
}

HyperlinkLabel[underline=false] {
    text-decoration: none;
}

HyperlinkLabel:hover {
    color: #007780;
}

HyperlinkLabel:pressed {
    color: #00a7b3;
}
FluentLabelBase{color:#ff000000}</string>
              </property>
              <property name="text">
               <string>历史消息长度压缩阈值</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="pixelFontSize" stdset="0">
               <number>-1</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="SpinBox" name="history_compress_limit_spin_box"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SmoothScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
